enhancements/etcd/automated-backups-no-config.md

```diff
@@ -0,0 +1,242 @@
+    Please find below this solution's Pros & Cons.
+    - Pros
+      - The PV will be bound to a specific master node.
+      - The Backup Pod will be scheduled always to this master node since it uses a PVC bound to this PV.
+      - Using an SSD is possible, since local volume allows mounting it into a Pod.
+      - Having the Backup Pod scheduling deterministic is vital, since the retention policy needs the recent backup in place to work.
+    - Cons
+      - If the master node where the PV is mounted became unhealthy/unavailable/unreachable etc.
+      - The backups are no longer accessible, also taking new backups is no longer possible.
+      - Ideally, the backups should be taken on a round-robin fashion on each master node to avoid overwhelming a specific etcd member more than others.
+      - Also spreading the backups among all etcd cluster members provide guarantees for disaster recovery in case of losing two members at the same time.
+      - However, using the `local volume` option over all the master nodes will be complicated and error prune.
+      
+
+### Final Storage Design proposal
+Best `Storage` solution is using `hybrid` approach, based on the OCP variant infrastructure.
+
+Based on the `infrastructure` resource type, we can create the storage programmatically according to the underlying infra.
+
+#### Cloud based OCP
+
+    Pros
+    - Utilising `Dynamic provisioning` is best option on cloud based OCP.
+    - Using the default `StorageClass` on the `PVC` manifest will provision a `PV`.
+    - The backups on the `PV` will be always available even if the master node is unavailable.
+    
+    Cons
+    -  There is no possibility to spread the backups over all the master nodes, since the `PV` is accessible within one and only one availability zone. 
+
+
+#### Single-node and Bare metal OCP
+    Only possible option is to use `local volume`. 
+
+### Open Questions
+How to use remote storage for backups on `BM` and `SNO`. 
+
+## Test Plan
+
+An e2e test will be added to practice the scenario as follows 
+
+- Enable AutomatedBackupNoConfig FeatureGate.
+- Verify that a `Etcdbackup` CR has been installed.
+- Verify that a PV, PVC has been created.
+- Verify that a backup has been taken.
+- Verify that the backups are valid according to the retention policy.
+
+## Graduation Criteria
+TBD
+
+### Dev Preview -> Tech Preview
+TBD
+
+### Tech Preview -> GA
+TBD
+
+### Removing a deprecated feature
+TBD
+
+## Upgrade / Downgrade Strategy
+TBD
+
+## Version Skew Strategy
+TBD
+
+## Operational Aspects of API Extensions
+TBD
+
+#### Failure Modes
+TBD
+
+## Support Procedures
+TBD
+
+## Implementation History
+TBD
+
+## Alternatives
+### hostPath
+
+Utilising `hostPath` volumes as a storage solution for the backups.
+
+`hostPath` mounts a path from the node's file system as a volume into the pod.
+
+It supports all OCP variants, including `SNO` and `BM`.  
+
+However, I am strongly against using it for the following reasons
```

Comment:
```suggestion
However, the following reasons suggest against that solution:
```

Reply:
this whole has been removed, radical changes :D :D